Overview.src.html

```diff
@@ -4841,6 +4996,11 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 getter must return <span title=concept-Request-request>request</span>'s
 <span title=concept-request-credentials-mode>credentials mode</span>.
 
+<p>The <dfn title=dom-Request-use-referred-token-binding><code>useReferredTokenBinding</code></dfn>
+attribute's getter must return true if <span title=concept-Request-request>request</span>'s
+<span title=concept-request-use-referred-token-binding>use-referred-token-binding flag</span>
+is set, and false otherwise.
```

Comment:
It seems like you did. I'd appreciate it if you could reply to the more substantial review comments so it's somewhat clear what's going on. That would help make reviewing easier, especially since you don't do updates all that often.